source/application/AMPIC887DriverServer/AMConsoleInputHandler.cpp

```diff
@@ -0,0 +1,64 @@
+#include "AMConsoleInputHandler.h"
+#include "AMAbstractConsoleCommandParser.h"
+
+AMConsoleInputHandler::AMConsoleInputHandler(AMAbstractConsoleCommandParser* commandParser,
+										   QObject* parent)
+	:QObject(parent),
+	  standardOutputStream_(stdout),
```

Comment:
I can change this. I tend to do this when one of the member variables is stored as a value and has no default parameterless constructor. I can change it, though I'm not sure if it'll involve me switching out the value member variables for references.